src/rf/apps/workers/management/commands/poll.py

```diff
@@ -3,13 +3,24 @@
 from __future__ import unicode_literals
 from __future__ import division
 
+import logging
+
 from django.core.management.base import BaseCommand
 
 from apps.workers import process
 
 
+log = logging.getLogger(__name__)
+
+
 class Command(BaseCommand):
-    def handle(self, *args, **options):
-        self.stdout.write('Polling started in worker queue.\n')
-        p = process.QueueProcessor()
-        p.start_polling()
+    def handle(self, *args, **kwargs):
+        log.info('Begin polling')
+        try:
+            p = process.QueueProcessor()
+            p.start_polling()
+        except KeyboardInterrupt:
```

Comment:
The main purpose of this is to avoid dumping a stack trace when you kill the poll task while running it interactively. I haven't thoroughly tested this interaction with the upstart job.